mdbenchmark/generate.py

```diff
@@ -26,53 +26,32 @@
 from jinja2.exceptions import TemplateNotFound
 
 from mdbenchmark import console
+from mdbenchmark.mdengines import gromacs, namd
 
 from .cli import cli
-from .utils import ENV, normalize_host, print_possible_hosts
+from .utils import ENV, detect_md_engine, normalize_host, print_possible_hosts
 
-
-def write_bench(top, tmpl, nodes, gpu, module, tpr, name, host, time):
-    sim = mds.Sim(
-        top['{}/'.format(nodes)],
-        categories={
-            'module': module,
-            'gpu': gpu,
-            'nodes': nodes,
-            'host': host,
-            'time': time,
-            'name': name,
-            'started': False
-        })
-
-    copyfile(tpr, sim[tpr].relpath)
-
-    # Add some time buffer to the requested time. Otherwise the queuing system
-    # kills the jobs before GROMACS can finish
-    formatted_time = '{:02d}:{:02d}:00'.format(*divmod(time + 5, 60))
-
-    # create bench job script
-    script = tmpl.render(
-        name=name,
-        gpu=gpu,
-        module=module,
-        n_nodes=nodes,
-        time=time,
-        formatted_time=formatted_time)
-
-    with open(sim['bench.job'].relpath, 'w') as fh:
-        fh.write(script)
+#import mdengines
 
 
 @cli.command()
 @click.option(
-    '-n', '--name', help='Name of .tpr file.', default='md', show_default=True)
+    '-n',
+    '--name',
+    help='Name of input files. All files must have the same base name.',
+    default='',
```

Comment:
Do we need `default=''` here? We can probably get rid of it

Reply:
yeah, this shouldn't have a default.